src/ReefGuideAPI.jl

```diff
@@ -55,26 +55,46 @@ Load regional data to act as an in-memory cache.
 OrderedDict of `RegionalCriteria` for each region.
 """
 function setup_regional_data(config::Dict)
+    reef_data_path = config["prepped_data"]["PREPPED_DATA_DIR"]
+
     if @isdefined(REGIONAL_DATA)
         @debug "Using previously generated regional data store."
         sleep(1)  # Pause so message is noticeably visible
+
+        reef_outline_path = joinpath(reef_data_path, "rrap_canonical_outlines.gpkg")
+        REGIONAL_DATA["reef_outlines"] = GDF.read(reef_outline_path)
+
+        REGIONAL_DATA["region_long_names"] = Dict(
+            "FarNorthern" => "Far Northern Management Area",
+            "Cairns-Cooktown" => "Cairns/Cooktown Management Area",
+            "Townsville-Whitsunday" => "Townsville/Whitsunday Management Area",
+            "Mackay-Capricorn" => "Mackay/Capricorn Management Area"
+        )
         return REGIONAL_DATA
     end
 
-    # Check disk-based store
+    # # Check disk-based store
     reg_cache_dir = config["server_config"]["REGIONAL_CACHE_DIR"]
     reg_cache_fn = joinpath(reg_cache_dir, "regional_cache.dat")
     if isfile(reg_cache_fn)
         @debug "Loading regional data cache from disk"
         @eval const REGIONAL_DATA = deserialize($(reg_cache_fn))
+
+        reef_outline_path = joinpath(reef_data_path, "rrap_canonical_outlines.gpkg")
```

Comment:
One issue I have here is that we'd have to rename the canonical reefs file every time there's an update, won't we?

Why not just use the original filename?

Reply:
The original file name has the date/time of creation in it so I thought it was easier to rename it when it is uploaded to the input data teams folder. Or should i use find_latest_file() to find it and use the original name?

Reply:
That would remove the need for us to manually do things, so I think use `find_latest_file()` (in fact, we should rename that to `find_latest_outlines()` or something more descriptive). It's an API-level utility too, so makes more sense to me for the function to live in the API rather than processing scripts.